storm-client/src/jvm/org/apache/storm/messaging/netty/Client.java

```diff
@@ -162,6 +163,8 @@ public class Client extends ConnectionWithStatus implements ISaslClient {
         launchChannelAliveThread();
         scheduleConnect(NO_DELAY_MS);
         int messageBatchSize = ObjectReader.getInt(topoConf.get(Config.STORM_NETTY_MESSAGE_BATCH_SIZE), 262144);
+        pendingMessagesFlushTimeoutMs = ObjectReader.getLong(topoConf.get(Config.STORM_MESSAGING_NETTY_FLUSH_TIMEOUT_MS));
```

Comment:
Should we define a default value here, if the config value is not (yet) updated (in legacy deployments for ex):

```
pendingMessagesFlushTimeoutMs = ObjectReader.getLong(topoConf.get(Config.STORM_MESSAGING_NETTY_FLUSH_TIMEOUT_MS, 600000L));
```